src/pages/api/definePassengerType.ts

```diff
@@ -144,12 +144,19 @@ export const passengerTypeDetailsSchema = yup
 export const formatRequestBody = (req: NextApiRequestWithSession): FilteredRequestBody => {
     const filteredReqBody: { [key: string]: string | string[] } = {};
     Object.entries(req.body).forEach(entry => {
-        if (
-            entry[0] === 'ageRangeMin' ||
-            entry[0] === 'ageRangeMax' ||
-            entry[0] === 'minNumber' ||
-            entry[0] === 'maxNumber'
-        ) {
+        if (entry[0] === 'minNumber' || entry[0] === 'maxNumber') {
+            const input = entry[1] as string;
+            const strippedInput = input.replace(/\s+/g, '');
+            if (strippedInput === '') {
+                return;
+            }
+            filteredReqBody[entry[0]] = strippedInput;
+            return;
+        }
+        if (entry[0] === 'ageRangeMin' || entry[0] === 'ageRangeMax') {
```

Comment:
Could we combine these? `if (entry[0] === 'ageRangeMin' || (entry[0] === 'ageRangeMax' && req.body.ageRange !== 'No')) {
            const input = entry[1] as string;
            const strippedInput = input.replace(/\s+/g, '');
            if (strippedInput === '') {
                return;
            }
            filteredReqBody[entry[0]] = strippedInput;
            return;
        }`

Reply:
Similar issue to the above comment, I've pushed up a simplified version again